examples/ego_open_agent.py

```diff
@@ -20,12 +20,13 @@
 AGENT_ID = "Agent-007"
 
 
-def main(scenarios, headless, num_episodes, seed):
+def main(scenarios, sim_name, headless, num_episodes, seed):
```

Comment:
nit:`experiment_name` might be more correct here.

Reply:
I saw a lot of places that use the word "simulation" in our code, is it supposed to be different from "experiment"?